README.md

```diff
@@ -1,2 +1,2 @@
 # hello-world
-optional
```

Comment:
Removed word optional from line 2 and replaced with line 3 content because duh
Added emoji of cat2 because you know